wolfcrypt/test/test.c

```diff
@@ -27921,13 +27921,15 @@ int mutex_test(void)
         return -12701;
     if (wc_LockMutex(&m) != 0)
         return -12702;
+#if !defined(WOLFSSL_SOLARIS)
     if (wc_FreeMutex(&m) != BAD_MUTEX_E)
         return -12703;
+#endif
     if (wc_UnLockMutex(&m) != 0)
         return -12704;
     if (wc_FreeMutex(&m) != 0)
         return -12705;
-#ifndef WOLFSSL_NO_MUTEXLOCK_AFTER_FREE
+#if !defined(WOLFSSL_NO_MUTEXLOCK_AFTER_FREE) && !defined(WOLFSSL_SOLARIS)
```

Comment:
Why is the SOLARIS define added here?  SOLARIS turns on NO_MUTEXLOCK_AFTER_FREE in settings.

Reply:
@toddouska, Removed.